lib/localization/locales/vi.json

```
@@ -0,0 +1,64 @@
+{
+  "loading": "Đang nạp...",
```

nạp is usually more charging a battery or adding

```suggestion
  "loading": "Đang tải...",
```

---

Charging the battery is "Đang sạc" (pin).

As for the decision of "nạp" instead of "tải", both technically means "load", but in the case "tải" it's more commonly used in upload/download in computing context, and "Đang tải/tải xuống" is a common short way to say "Downloading" (and for uploading it's "tải lên" since it's less common)

Hence the usage of "nạp" to distinct "Loading" instead of "Downloading".

---

ahhh, that makes sense, thanks for the explanation!